modules/claims_api/spec/lib/claims_api/document_service_base_spec.rb

```diff
@@ -9,22 +9,28 @@
   let(:claim_id) { '581128c6-ad08-4b1e-8b82-c3640e829fb3' }
   let(:file_number) { '123456789' }
   let(:participant_id) { '987654321' }
+  let(:dependent_veteran_name) { 'margie_curtis' }
+  let(:ews_suffix) { '5103' }
+
+  let(:ews_doc_type) { 'L705' }
+  let(:dis_comp_doc_type) { 'L122' }
+  let(:correspondence_doc_type) { 'L023' }
```

Comment:
Some of this is just refactoring to cut down on some duplication.  There is a new test added at the very bottom of this file for the new naming scenario